src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -1355,6 +1368,17 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 			{
 				AiNodeSetFlt( node, g_velocityOutlierThresholdArnoldString, velocityOutlierThreshold.get() );
 			}
+
+			if ( stepSize )
+			{
+				AiNodeSetFlt( node, g_stepSizeArnoldString, stepSize.get() );
```

Comment:
I thought in this case we were going to multiply in the step scale too, because we felt that step scale should be applied consistently, regardless of step_size?